R/correctionsataglance-data.R

```diff
@@ -101,13 +101,14 @@ formatDateRange <- function(startD, endD){
     endSeq <- c(startSeq[-1], endD)
   }
   
-  #don't print Month Year in plot if there isn't enough room inside the rectangle
+#   #don't print Month Year in plot if there isn't enough room inside the rectangle
   dateSeq <- startSeq
   startSpan <- as.numeric(endSeq[1] - startSeq[1])
   endSpan <- as.numeric(tail(endSeq, 1) - tail(startSeq, 1))
-  if(startSpan <= 16){dateSeq[1] <- NA}
-  if(endSpan <= 14){dateSeq[length(dateSeq)] <- NA}
-  
+  startLabelText <- format(startD, "%b %Y")
+  endLabelText <-  format(endD, "%b %Y")
+  #if(isTextLong(labelText=startLabelText,dateLim=NULL,startD,endD,totalDays=numdays)) {dateSeq[1] <- NA}
```

Comment:
Isn't dateSeq the labels for the rectangles (poor naming, it should be more clear - sorry that was my fault!)? I think the points is to make any label that is bigger than the rectangle not show up. If startD and endD are vectors, I think you might want to do this?

```r
dateSeq[isTextLong(labelText=startLabelText,dateLim=NULL,startD,endD,totalDays=numdays)] <- NA
```